api/scripts/switch-campaign-to-flash.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+const _ = require('lodash');
+require('dotenv').config();
+const { knex } = require('../db/knex-database-connection');
+
+async function switchCampaignToFlash(id) {
+  await knex('campaigns').update({ assessmentMethod: 'FLASH' }).where({ id });
+}
+
+async function main(id) {
+  if (!_.isFinite(id)) {
+    throw new Error("L'id de campagne est obligatoire, node switch-campaign-to-flash.js <:id>");
+  }
+
+  try {
+    await switchCampaignToFlash(id);
+    console.log(`Campaign ${id} has successfully been updated`);
+  } catch (err) {
+    console.log(err);
+  }
+}
+
+if (require.main === module) {
+  const id = parseInt(process.argv[2]);
+
+  main(id).then(
+    () => process.exit(0),
+    (err) => {
+      console.error(err);
+      process.exit(1);
+    }
+  );
```

Comment:
Il n'est pas necessaire de faire un process.exit, comme suggere par le linter
```suggestion
    main(id).catch(err => {
       console.error(err);
       process.exitCode = 1; // ou throw err
    })

    }
  );
```